src/main/java/gov/cabinetoffice/gap/adminbackend/utils/XlsxGenerator.java

```diff
@@ -51,6 +51,9 @@ private static void addHeaders(Sheet worksheet, List<String> headers) {
         for (int col = 0; col < headers.size(); col++) {
             Cell cell = row.createCell(col);
             cell.setCellValue(headers.get(col));
+
+            // leaving this method call in to apply basic formatting with minimal
+            // performance hit
```

Comment:
could we run `addHeaders` after `addData` to still get accurate column sizes, but without running it `worksheet width / worksheet height` times?

Reply:
This is a good shout, will give it a go just now and see how I get on.

Reply:
This worked a treat. Cheers for the suggestion!